bundle-size/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "dev": "nodemon",
-    "deploy": "gcloud app deploy",
+    "deploy": "gcloud --project amp-bundle-size-bot app deploy",
```

Comment:
Does this work? According to the [docs](https://cloud.google.com/sdk/gcloud/reference/app/deploy), it appears that the `--project` option should appear after `gcloud app deploy`.

Reply:
Yep, it works. I prefer to put flags before positional arguments (== opposite of how most languages accept arguments, but whatever 😝)